### ISSUE TITLE
occasionally file action confirmation dialog get stuck

### ISSUE THREAD

**Issue by individual-it** [owncloud/web#1728]
### Steps to reproduce
1. delete or rename file
2. get unlucky

### Expected behaviour
dialog should close

### Actual behaviour
sometimes it get stuck for me after confirmation
![image](https://user-images.githubusercontent.com/2425577/62680286-9a0a0a80-b9d6-11e9-8323-3eb31e50d69f.png)

### Server configuration
**Operating system**:
Ubuntu 18.04
**Web server:**
Apache 2
**Database:**
MySQL
**PHP version:**
PHP 7.1
**ownCloud version:** (see ownCloud admin page)
10.3.0 prealpha (git) 
**Updated from an older ownCloud or fresh install:**
fresh
**Where did you install ownCloud from:**
git
**The content of config/config.php:**

```
{
    "system": {
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "localhost",
            "172.17.0.1"
        ],
        "datadirectory": "\/home\/artur\/www\/owncloud-core\/data",
        "overwrite.cli.url": "http:\/\/localhost\/owncloud-core\/",
        "version": "10.3.0.0",
        "logtimezone": "UTC",
        "htaccess.RewriteBase": "\/owncloud-core\/",
        "ldapIgnoreNamingRules": false,
        "loglevel": 1,
        "log_type": "owncloud",
        "dbname": "owncloud_20190701",
        "dbhost": "localhost",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "theme": "",
        "sharing.federation.allowHttpFallback": true,
        "files_external_allow_create_new_local": "true",
        "license-key": "***REMOVED SENSITIVE VALUE***",
        "instanceid": "oc4kyqahfanl",
        "apps_paths": [
            {
                "path": "\/home\/artur\/www\/owncloud-core\/apps",
                "url": "apps",
                "writable": true
            },
            {
                "path": "\/home\/artur\/www\/owncloud-core\/apps2",
                "url": "apps2",
                "writable": true
            }
        ],
        "dbtype": "mysql",
        "allow_user_to_change_display_name": false,
        "csrf.disabled": true,
        "enable_previews": false,
        "maintenance": false,
        "cors.allowed-domains": [
            "http:\/\/172.17.0.1:8300"
        ],
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "skeletondirectory": "\/home\/artur\/www\/owncloud-core\/apps\/testing\/data\/webUISkeleton"
    }
}

```

**List of activated apps:**

```
Enabled:
  - activity: 2.4.2
  - comments: 0.3.0
  - dav: 0.4.0
  - diagnostics: 0.1.3
  - enterprise_key: 0.1.4
  - federatedfilesharing: 0.5.0
  - federation: 0.1.0
  - files: 1.5.2
  - files_external: 0.7.1
  - files_sharing: 0.12.0
  - files_texteditor: 2.3.0
  - files_trashbin: 0.9.1
  - files_versions: 1.3.0
  - firstrunwizard: 1.2.0
  - notifications: 0.5.0
  - oauth2: 0.4.0
  - provisioning_api: 0.5.0
  - systemtags: 0.3.0
  - systemtags_management: 0.3.1
  - testing: 0.1.0
  - updatenotification: 0.2.1
  - user_management: 0.0.1
  - windows_network_drive: 0.7.2
  - workflow: 0.3.0
Disabled:
  - admin_audit
  - brute_force_protection
  - customgroups
  - encryption
  - files_antivirus
  - files_classifier
  - files_primary_s3
  - files_lifecycle
  - firewall
  - guests
  - password_policy
  - ransomware_protection
  - richdocuments
  - search_elastic
  - sharepoint
  - twofactor_totp
  - user_ldap
  - wopi
```

**Are you using external storage, if yes which one:** local/smb/sftp/...
no
**Are you using encryption:** yes/no
no
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no

### Client configuration
**Browser:**
FF 68
**Operating system:**
Ubuntu 18.04

#### Browser log
```
Unhandled promise rejection 
Object { _name: "NavigationDuplicated", name: "NavigationDuplicated", stack: "" }
files.bundle.js:15:98639
    exports http://172.17.0.1:8300/apps/files/files.bundle.js:15
    q http://172.17.0.1:8300/apps/files/files.bundle.js:7
    n http://172.17.0.1:8300/apps/files/files.bundle.js:7
    exports http://172.17.0.1:8300/apps/files/files.bundle.js:15
    Q http://172.17.0.1:8300/apps/files/files.bundle.js:7
    <anonymous> http://172.17.0.1:8300/core/core.bundle.js:55
    b http://172.17.0.1:8300/core/core.bundle.js:55
    x http://172.17.0.1:8300/core/core.bundle.js:55
```

**Comment by DeepDiver1975** [comment]
@LukasHirt you have been working in this area already - right?

**Comment by LukasHirt** [comment]
@DeepDiver1975 I was fixing the one for overwriting files while uploading. There it was caused by different issue though since looking at the screenshot this at least deletes the message of the dialog...

**Comment by PVince81** [comment]
same kind of issue but where the button stays disabled: https://github.com/owncloud/phoenix/issues/2027

**Comment by PVince81** [comment]
seems many people are getting unlucky now: tests/acceptance/features/webUITrashbin/trashbinFilesFolders.feature:13  is failing on multiple PRs.

Raising priority to p1 as this will disturb dev

**Comment by skshetry** [comment]
@PVince81, the "workaround" was merged today (#2417). So, priority can be reduced from `p1`, I guess, so, please triage.

## Analysis:
I can revisit this later, and try to see what the real problem is. It could be a multitude of following issues:
1. Timing (so, add more waits, which does seem to fix the issue)
2. Selenium received invalid requests for click / Didnot receive at all, due to `Nightwatch` (probable,
as there are many issues regarding this).
3. Something is blocking "click" (highly probable)
4. Maybe, it is working after all. The popup might be taking time to disappear.

We fixed this by `retrying` the click action. But, requires more investigation. I could only reproduce in
`zalenium` as it is too slow (and, that too running it in `-m 150m --cpus="0.5"` and two browsers already opened).

**Comment by LukasHirt** [comment]
This should be fixed in this PR #3378. I have removed in that PR the workaround in the tests so we need to keep an eye out after it gets merged if it'll be passing. For now, keeping this still open

**Comment by pascalwengerter** [comment]
Closing this since we've introduced a new modal component a while ago and I've never heard about anyone getting stuck with it 😇
